hack/get-manifest-ds.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+DIRNAME="$(dirname "$(readlink -f "$0")")"
+DEFAULT_IMAGE="quay.io/k8stopologyawarewg/resource-topology-exporter:latest"
+REPOOWNER=${REPOOWNER:-k8stopologyawarewg}
+IMAGENAME=${IMAGENAME:-resource-topology-exporter}
+IMAGETAG=${IMAGETAG:-latest}
+export RTE_CONTAINER_IMAGE=${RTE_CONTAINER_IMAGE:-quay.io/${REPOOWNER}/${IMAGENAME}:${IMAGETAG}}
+envsubst < ${DIRNAME}/../manifests/resource-topology-exporter-ds.yaml
```

Comment:
What env variable you're trying to substitute here?

Reply:
it's `RTE_CONTAINER_IMAGE`: https://github.com/k8stopologyawareschedwg/resource-topology-exporter/blob/master/manifests/resource-topology-exporter-ds.yaml#L45